chapter09/tests/dags/02_test_example.py

```diff
@@ -0,0 +1,9 @@
+from airflow.operators.bash import BashOperator
```

Comment:
This test is not automatically resolved by pytest. You need to name it test_something.py